stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/BaseMessageItemViewHolder.kt

```diff
@@ -1,19 +1,33 @@
 package io.getstream.chat.android.ui.messages.adapter
 
 import android.view.View
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.constraintlayout.widget.ConstraintSet
 import androidx.recyclerview.widget.RecyclerView
 import com.getstream.sdk.chat.adapter.MessageListItem
 import io.getstream.chat.android.ui.messages.adapter.viewholder.decorator.BackgroundDecorator
 import io.getstream.chat.android.ui.messages.adapter.viewholder.decorator.Decorator
 import io.getstream.chat.android.ui.messages.adapter.viewholder.decorator.GapDecorator
+import io.getstream.chat.android.ui.messages.adapter.viewholder.decorator.MaxPossibleWidthDecorator
 
 public abstract class BaseMessageItemViewHolder<T : MessageListItem>(itemView: View) : RecyclerView.ViewHolder(itemView) {
-    private val decorators = listOf<Decorator>(BackgroundDecorator(), GapDecorator())
+    private val decorators = listOf<Decorator>(BackgroundDecorator(), GapDecorator(), MaxPossibleWidthDecorator())
 
     public fun bind(data: T) {
         decorators.forEach { it.decorate(this, data) }
         bindData(data)
     }
 
     public abstract fun bindData(data: T)
+
+    protected fun constraintView(isMine: Boolean, view: View, layout: ConstraintLayout) {
+        ConstraintSet().apply {
+            clone(layout)
+            clear(view.id, ConstraintSet.LEFT)
+            clear(view.id, ConstraintSet.RIGHT)
+            val anchorSide = if (isMine) ConstraintSet.END else ConstraintSet.START
```

Comment:
We have a mix of LEFT/RIGHT and START/END here, which seems weird.

Reply:
We should use LEFT/RIGHT right now.